### ISSUE TITLE
Lack of keyboard focus on engine dropdown

### ISSUE THREAD

**Issue by CorrineS** [LibreChat-AI/librechat.ai#316]
Keyboard focus is present except when focus is placed model choice:

Missing keyboard focus indicator
![Image](https://github.com/user-attachments/assets/f7ec64df-418a-48c2-a94d-4c8bcdbfbd21)

Keyboard focus indicator for others:
![Image](https://github.com/user-attachments/assets/640a4e39-16af-4f89-8a73-c383358b7d34)

Focus seems to be present in the CSS but something is blocking its perception
